src/app/components/flow-chart-editor/flow-chart-editor.component.ts

```diff
@@ -145,7 +148,7 @@ export class FlowChartEditorComponent {
         x2: edge.line.endPoint.x, y2: edge.line.endPoint.y,
         selected: this.selectionInModel.isEdgeHighlightedInDrawing(edge.getKey(), this.layoutModel!),
         arrow: edge.isLastSegment,
-        isError: edge.optionalOriginalText === 'error'
+        isError: FlowChartEditorComponent.errorForwardNames.includes(edge.optionalOriginalText||'success') || edge.getFrom().originalStyle === 'errorOutline'
       }})
     this.drawing = {width: layout.width, height: layout.height, rectangles, lines}
```

Comment:
Is deze styling ook relevant voor de statische SVG? In dat geval stel ik voor dat we deze logica verplaatsen naar Layout. Dan wordt het misschien ook makkelijker om de Karma test die ik hierboven vraag te maken. Het idee van Drawing is dat de informatie uit de Layout opnieuw wordt geordend zodat je er makkelijker bij kan in Angular HTML files. Het heeft mijn voorkeur als complexe logica om informatie te genereren gebeurt bij het maken van de Layout.

Eigenlijk zou Layout wat mij betreft makkelijker moeten zijn om te gebruiken in Angular HTML files. Ik wist toen ik code begon te maken niet hoe je in TypeScript met velden kan omgaan, daarom zijn er getters in plaats van gewoon velden. Het zou mooi zijn om dat een keer in een aparte PR te corrigeren. Misschien kan Drawing dan helemaal weg.